charter.md

```diff
@@ -22,14 +22,19 @@ Server-initiated services are out of scope. The working group will first deliver
 a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
 IP proxying.
 
-The working group will consider fallback to versions of HTTPS that operate over TCP as a mitigation to UDP or HTTP/3
-blocking. Moreover, the working group will consider implications of tunneling protocols with congestion control and loss
-recovery over MASQUE, and may issue recommendations accordingly. New congestion control and loss recovery algorithms are
+Exercising the extension points defined by CONNECT-UDP and CONNECT-IP is important to ensure their continued useability.
+The working group will consider extensions to these documents that provide missing functionality or enhance performance
+for real world use cases. The initial set of extensions will be in support of UDP listening, QUIC-aware proxying, and
+datagram priorities. Extensions that help enable CONNECT-UDP or CONNECT-IP services to coordinate or work well in the
+ecosystem of other services may be considered. Extensions to HTTP/3 Datagrams will be coordinated with HTTPBIS.
+Extensions that solely relate to generic proxying functionality, and are not specific to the core MASQUE documents, are
 out of scope.
 
-Multicast support is out of scope. However, the group may specify extension points that would enable future work on
-multicast. Specifying proxy server discovery mechanisms is also out of scope. However, the group may consider features,
-such as proxy server identifiers, that aid future discovery mechanisms.
+The working group will consider fallback to versions of HTTPS that operate over TCP as a mitigation to UDP or HTTP/3
```

Comment:
Versions of HTTP, not HTTPS?

Reply:
Yeah, this was in there previously as 
> The working group will consider fallback to versions of HTTPS that operate over TCP as a mitigation to UDP or HTTP/3

but seems good to clean up something this small either way

Reply:
I'm OK with "versions of HTTP" for this 👍

Reply:
+1

Reply:
Updated in [8af81ca](https://github.com/ietf-wg-masque/wg-materials/pull/7/commits/8af81ca9873dc5f769a87707af732f24be6455bf)